donkeycar/templates/donkey2.py

```diff
@@ -41,7 +40,7 @@ def drive(cfg, model_path=None, use_joystick=False, use_chaos=False):
     V = dk.vehicle.Vehicle()
 
     clock = Timestamp()
-    V.add(clock, outputs='timestamp')
+    V.add(clock, outputs=['timestamp'])
```

Comment:
Nice catch.
With this fix the timestamp is properly saved to json file.

Previously the `.json` file looks like (`timestamp` is `null`):
> {"user/angle": -0.6392101809747612, "timestamp": null, "cam/image_array": "18500_cam-image_array_.jpg", "user/throttle": 0.63, "user/mode": "user"}